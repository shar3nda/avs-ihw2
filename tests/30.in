},k<O/_KhD)3k-DQQWn-}q}_ixSRGB4IBitwi'-vh6E!}";pJo%\&?{C,C;,;g^FtTm&93EV@T%V?Mlxa