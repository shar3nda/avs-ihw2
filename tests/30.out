Lowercase: 21, uppercase: 24